src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,629 @@
+  'Egypt',
+  'El Salvador',
+  'England',
+  'Equatorial Guinea',
+  'Eritrea',
+  'Estonia',
+  'Ethiopia',
+  'Fiji',
+  'Finland',
+  'France',
+  'French Guiana',
+  'Gabon',
+  'Gambia',
+  'Georgia',
+  'Germany',
+  'Ghana',
+  'Gibraltar',
+  'Great Britain',
+  'Great Britain and Gibraltar',
+  'Greece',
+  'Greenland',
+  'Grenada',
+  'Guadeloupe',
+  'Guatemala',
+  'Guinea',
+  'Guinea, Republic of Guinea',
+  'Guinea-Bissau',
+  'Guyana',
+  'Haiti',
+  'Honduras',
+  'Hong Kong',
+  'Hungary',
+  'Iceland',
+  'India',
+  'Indonesia',
+  'Iran',
+  'Iraq',
+  'Ireland',
+  'Israel (Jerusalem)',
+  'Israel (Tel Aviv)',
+  'Italy',
+  'Jamaica',
+  'Japan',
+  'Jordan',
+  'Kazakhstan',
+  'Kenya',
+  'Kosovo',
+  'Kuwait',
+  'Kyrgyzstan',
+  'Laos',
+  'Latvia',
+  'Lebanon',
+  'Leeward Islands',
+  'Lesotho',
+  'Liberia',
+  'Libya',
+  'Liechtenstein',
+  'Lithuania',
+  'Luxembourg',
+  'Macao',
+  'Macedonia',
+  'Madagascar',
+  'Malawi',
+  'Malaysia',
+  'Mali',
+  'Malta',
+  'Martinique',
+  'Mauritania',
+  'Mauritius',
+  'Mexico',
+  'Moldavia',
+  'Mongolia',
+  'Montenegro',
+  'Montserrat',
+  'Morocco',
+  'Mozambique',
+  'Namibia',
+  'Nepal',
+  'Netherlands',
+  'Netherlands Antilles',
+  'Nevis',
+  'New Caledonia',
+  'New Zealand',
+  'Nicaragua',
+  'Niger',
+  'Nigeria',
+  'North Korea',
+  'Northern Ireland',
+  'Norway',
+  'Oman',
+  'Pakistan',
+  'Panama',
+  'Papua New Guinea',
+  'Paraguay',
+  'Peru',
+  'Philippines',
+  'Philippines (restricted payments)',
+  'Poland',
+  'Portugal',
+  'Qatar',
+  'Republic of Yemen',
+  'Romania',
+  'Russia',
+  'Rwanda',
+  'Sao-Tome/Principe',
+  'Saudi Arabia',
+  'Scotland',
+  'Senegal',
+  'Serbia',
+  'Serbia/Montenegro',
+  'Seychelles',
+  'Sicily',
+  'Sierra Leone',
+  'Singapore',
+  'Slovakia',
+  'Slovenia',
+  'Somalia',
+  'South Africa',
+  'South Korea',
+  'Spain',
+  'Sri Lanka',
+  'St. Kitts',
+  'St. Lucia',
+  'St. Vincent',
+  'Sudan',
+  'Suriname',
+  'Swaziland',
+  'Sweden',
+  'Switzerland',
+  'Syria',
+  'Taiwan',
+  'Tajikistan',
+  'Tanzania',
+  'Thailand',
+  'Togo',
+  'Trinidad and Tobago',
+  'Tunisia',
+  'Turkey (Adana only)',
+  'Turkey (except Adana)',
+  'Turkmenistan',
+  'Uganda',
+  'Ukraine',
+  'United Arab Emirates',
+  'United Kingdom',
+  'Uruguay',
+  'Uzbekistan',
+  'Vanuatu',
+  'Venezuela',
+  'Vietnam',
+  'Wales',
+  'Western Samoa',
+  'Yemen Arab Republic',
+  'Zambia',
+  'Zimbabwe'
+];
+
+export const militaryPostOfficeTypeLabels = { // TODO: determine whether these are necessary
+  APO: 'Army Post Office',
+  FPO: 'Fleet Post Office',
+  DPO: 'Diplomatic Post Office'
+};
+
+const addressUISchema = (addressName, title) => {
+  return {
+    'ui:title': title,
+    type: {
+      'ui:title': 'Type',
+      'ui:options': {
+        labels: typeLabels
+      }
+    },
+    country: {
+      'ui:title': 'Country',
+      'ui:options': {
+        labels: {
+          USA: 'United States of America'
```

Comment:
@markgreenburg the label change has been made here.

Reply:
Should `src/platform/forms/address.js` be used instead? I see pciu address is also included there.

Reply:
This will be changed to "United States" as that is used elsewhere.